### PR TITLE
Change of hosting for the static site

### PR DESCRIPTION
**Pull request by mattempty**
Deployment and Dockerfile changed from running the Middleman dev server to being a httpd server and deployment script updated to latest command.

### CODE REVIEW

**Comment by tomas-stefano** [comment]
Can the build directory be ignored on .gitignore?

File: Dockerfile [hunk 1]
```diff
@@ -1,20 +1,19 @@
-FROM ministryofjustice/ruby:2.5.3-webapp-onbuild
+FROM httpd:2.4.43-alpine
```
Comment (brenetic) [inline Dockerfile:1]: how come we need to drop the version?

Comment (mattempty) [reply Dockerfile:1]: 2.4.43 is the latest alpine version available in the docker httpd image. https://hub.docker.com/_/httpd

Comment (brenetic) [reply Dockerfile:1]: i think we will need to find a way around this. 2.4 is pretty old and we should really be using 2.7.1

Comment (brenetic) [reply Dockerfile:1]: oh wait! ignore me. i was thinking it was the ruby version 😆 ! this is fine!

**Comment by brenetic** [comment]
can we remove the build folder that was checked in please

**Pull request merged**